.changeset/afraid-rocks-design.md

```diff
@@ -2,7 +2,7 @@
 "@replayio/test-utils": patch
 ---
 
-pr: #647
```

Comment:
we didn't yet reach such numbers here ;p not sure where it came from 😉

Reply:
Prob PRO-647